source/docs/ROS/GettingStarted/installing-ros.rst

```diff
@@ -0,0 +1,52 @@
+Installing ROS
+==============
+
+Downloading the Base Repo
+-------------------------
+
+To get started make sure ROS is installed on your machine:
+
+If you haven't already, run ``installNoetic.sh`` to install ROS Noetic. For more information, visit `ROS Noetic <http://wiki.ros.org/noetic/Installation/Debian>`__
+
+.. code-block:: rst
+   
+   ./installNoetic.sh
+
+.. important:: Running the ``installNoetic.sh`` script includes all the required tools and dependencies needed for the ROS Package, however this will require some time as the process can take approximately 3 hours. 
+
```

Comment:
Change this section to installing ROS manually without using the provided image

Reply:
I'll add instructions on installing ROS both manually an using the image

Reply:
Installing the image would not be installing ROS Manually, so the section title should just be ROS setup. 

I think the ROS Manual section should still be here though, with the installNoetic script.